addons/event/views/event_views.xml

```diff
@@ -714,8 +732,15 @@
             <field name="name">Attendees</field>
             <field name="res_model">event.registration</field>
             <field name="domain"></field>
-            <field name="view_mode">pivot,graph,kanban,tree,form</field>
-            <field name="context">{}</field>
+            <field name="view_mode">graph,pivot,kanban,tree,form</field>
+            <field name="context">{
+                    'search_default_filter_last_month_creation': 1,
+                    'search_default_status': 1,
+                    'search_default_group_by_create_date_day': 1,
+                    'search_default_group_event': 1,
+                    'attendees_from_reporting': 1,
```

Comment:
Mmmh ... ``registration_view_hide_group_by_create_date_week`` maybe ? A bit long ... but more explicit. Don't like the "from reporting" as it may be used outside of reporting (action used in other context).